### ISSUE TITLE
false alert on file src/server.js

### ISSUE THREAD

**Issue by jfoclpf** [depcheck/depcheck#655]
depcheck says I don't use `util`, but it's right there at the top of the file `src/server.js`

```
joao@toshiba:~/dev/autocosts$ npx depcheck --ignore-bin-package=true
Unused dependencies
* ajv
* chart.js
* jquery
* jquery-color
* jquery-sidebar
* path
* pdfmake
* smart-app-banner
* util
Unused devDependencies
* libxmljs
Missing dependencies
* chrome-launcher: ./test/googleLighthouse.js
* system: ./build/rasterTables.js
joao@toshiba:~/dev/autocosts$ cat src/server.js | grep util
const util = require('util')
debug(util.inspect(serverData, { showHidden: false, depth: null }))
  debug(util.inspect(serverData, { showHidden: false, depth: null }))
```

**Comment by jfoclpf** [comment]
another very strange false alarm, module `path` which I use everywhere, depcheck oddly says I don't use it

```joao@toshiba:~/dev/autocosts$ npx depcheck
Unused dependencies
* ajv
* path
* util
Unused devDependencies
* csslint
* libxmljs
* mocha
* webpage
joao@toshiba:~/dev/autocosts$ grep -R require\(\'path\'\) src/* test/*
src/server/webmanifest.js:const path = require('path')
src/server/statsCC.js:const path = require('path')
src/server/domains.js:const path = require('path')
src/server/list.js:const path = require('path')
src/server/preprocess.js:const path = require('path')
src/server/worldStats.js:const path = require('path')
src/server/sitemap.js:const path = require('path')
src/server/getUBER.js:const path = require('path')
src/server/renderPageCC.js:const path = require('path')
src/server.js:const path = require('path')
test/googleLighthouse.js:const path = require('path')
test/testApi.js:const path = require('path')
test/test.js:const path = require('path')
test/testServer.js:const path = require('path')
test/validateCalculator.js:const path = require('path')
test/validateCss.js:const path = require('path')
test/validateHtml.js:const path = require('path')
test/validateHtmlW3C.js:const path = require('path')
test/validateJs.js:const path = require('path')


```

**Comment by jfoclpf** [comment]
AHH, now I get it :) there's no need to install `path`
https://nodejs.dev/learn/the-nodejs-path-module

>There is no need to install it. Being part of the Node.js core, it can be used by simply requiring it:
>
>`const path = require('path')`

and the same for `util` I guess


You guys are great!
